--- a/setup.py
+++ b/setup.py
@@ -59,8 +59,9 @@
     params = py2exe_params
 else:
     files_spec = [
-        ('etc/bash_completion.d', ['youtube-dl.bash-completion']),
-        ('etc/fish/completions', ['youtube-dl.fish']),
+        ('../etc/bash_completion.d', ['youtube-dl.bash-completion']),
+        ('../etc/fish/completions', ['youtube-dl.fish']),
+        ('share/zsh/site-functions', ['youtube-dl.zsh']),
         ('share/doc/youtube_dl', ['README.txt']),
         ('share/man/man1', ['youtube-dl.1'])
     ]
